boltzrpc/boltzrpc.proto

```diff
@@ -733,6 +738,39 @@ message GetWalletRequest {
     optional uint64 id = 2;
 }
 
+message ListWalletTransactionsRequest {
+    uint64 id = 1;
+    optional bool exclude_swap_related = 2;
+}
+
+enum TransactionType {
+    ANY = 0;
+    LOCKUP = 1;
+    REFUND = 2;
+    CLAIM = 3;
+    RECEIVE = 4;
+    SEND = 5;
+}
+
+message WalletTransaction {
+    string tx_id = 1;
+    uint64 total_amount = 2;
+    int64 timestamp = 3;
+    repeated TransactionOutput outputs = 4;
+    uint32 block_height = 5;
+    optional string swap_id = 6;
+    TransactionType type = 7;
```

Comment:
What if we ever do batching; multiple swap related actions in a single transaction?

Reply:
we can make it repeated. the type can stay the same unless we start to mix claims and refunds together